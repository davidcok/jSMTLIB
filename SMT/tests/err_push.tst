; tests that the argument to a push is a number
( set-logic QF_UF)
( push 0.0 )
( push "asd")
( push #b1 )
( push #x0 )
( push a )
( push |A| )
( push 1 2 3 )
( push -1 )
(push)
