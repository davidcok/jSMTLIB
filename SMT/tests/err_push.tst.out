success
( push 0.0 )
       ^^^
(error "Expected a numeral here, instead of a decimal")
( push "asd")
       ^^^^^
(error "Expected a numeral here, instead of a string-literal")
( push #b1 )
       ^^^
(error "Expected a numeral here, instead of a binary")
( push #x0 )
       ^^^
(error "Expected a numeral here, instead of a hex-literal")
( push a )
       ^
(error "Expected a numeral here, instead of a symbol")
( push |A| )
       ^^^
(error "Expected a numeral here, instead of a symbol")
( push 1 2 3 )
         ^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
( push -1 )
       ^^
(error "Expected a numeral here, instead of a symbol")
(push)
     ^
(error "Expected a numeral here, instead of a )")
